app.py

```diff
@@ -19,13 +17,10 @@ def create_app(testing=False) -> Flask:
 
     flask_app = connexion_app.app
 
-    if (os.environ.get("FLASK_ENV") == "development") | testing:
+    if testing:
         flask_app.config.from_object(
             "config.environments.unit_testing.UnitTestingConfig"
         )
-        from config.environments.unit_testing import UnitTestingConfig
-
-        UnitTestingConfig.pretty_print()
```

Comment:
🔧 I think this can be changed so that we set testing in pytest.ini instead and the corresponding config is loaded.

Reply:
💯 have included the latest config set-up via pytest.ini 👍